packages/create-cloudflare/templates/hono/snippets/bindingsType.ts

```diff
@@ -0,0 +1 @@
+type Bindings = Record<string, unknown> & CloudflareBindings;
```

Comment:
```suggestion
type Bindings = Record<string, unknown> & CloudflareBindings;

```

Reply:
Also I think hono in its template index.ts does not use semicolons:
```suggestion
type Bindings = Record<string, unknown> & CloudflareBindings
```

Yours being the only one present:
![Screenshot 2024-04-22 at 10 12 00](https://github.com/cloudflare/workers-sdk/assets/61631103/78015f68-64cd-473b-aca8-8e720e3d8049)